libpysal/weights/tests/test_contiguity.py

```diff
@@ -109,6 +117,8 @@ def test_from_geodataframe_order(self):
             assert w.id_order[:5] == expected
 
     def test_from_xarray(self):
+        _  = pytest.importorskip("xarray")
```

Comment:
From what I understand here don't need to actually declare anything, we can have `pytest.importorskip("xarray")`